docs/on_chain_spec.md

```diff
@@ -467,8 +467,8 @@ bidderSignatureMessage
   -> BuiltinByteString
 bidderSignatureMessage auctionId bidPrice bidderPkh =
   toByteString auctionId <>
-  toByteString bidPrice <>
-  toByteString bidderPkh
+  toByteString bidderPkh <>
```

Comment:
Why?

Reply:
Currency symbol and pkh are more likely to be fixed size, while bid price byte length varies by user.
Less likely to be abusable, if we put the price at the end.